source/virtualBuffers/gecko_ia2.py

```diff
@@ -238,6 +264,35 @@ class Gecko_ia2(VirtualBuffer):
 	#: frame/iframe in the lists is a tuple of (IAccessible2_2, uniqueId). This
 	#: cache is used across instances.
 	_framesCache = weakref.WeakKeyDictionary()
+	_nativeAppSelectionMode = False
+
+	@script(
+		gesture="kb:NVDA+shift+f10"
+	)
+	def script_toggleNativeAppSelectionMode(self, gesture):
+		self._nativeAppSelectionMode = not self._nativeAppSelectionMode
+		if self._nativeAppSelectionMode:
+			ui.message(_("Native app selection mode enabled."))
+			try:
+				self.updateAppSelection()
+			except NotImplementedError:
+				pass
+		else:
+			ui.message(_("Native app selection mode disabled."))
+			try:
+				self.clearAppSelection()
+			except NotImplementedError:
+				pass
+
+	@script(
+		gesture="kb:control+c"
+	)
+	def script_copyToClipboard(self, gesture):
+		if self._nativeAppSelectionMode:
+			ui.message(_("native copy"))

```

Comment:
this needs a translators comment

Reply:
Does a user understand what "Native copy" means? Since native copy mode is opt in for now, I guess yes. But  what if one day the native copy mode becomes the default one?
Long time screen reader users are used to the copy on a web page being plain text. But I guess new users, especially ones being sighted before expect that the copy works as they did with the mouse, i.e. copy formatted text.

Reply:
@michaelDCurran - thoughts on naming this?
Ideas:
- application copy
- rich copy
- formatted copy

Reply:
See cyrilles above comment for reasoning